tests/unit/utils/test_stream.py

```diff
@@ -18,9 +18,8 @@ def agent(self, request):
         return Agent(stream=request.param, tools=[MockTool()], prompt_driver=driver)
 
     def test_init(self, agent):
+        chat_stream = Stream(agent)
         if agent.stream:
```

Comment:
Do you even need this branching at all anymore? Seems like both branches do almost thing. -> one just calls next twice?

Reply:
If nothing else, to keep code coverage happy, yes.